S/SLICOT/build_tarballs.jl

```diff
@@ -103,7 +119,8 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93"); compat="3.0.4"),
+    Dependency(get_addable_spec("libblastrampoline_jll", v"3.0.4+0"); platforms=filter(p -> VersionNumber(p["julia_version"]) == v"1.7.0", platforms)),
```

Comment:
Hmm, this line doesn't seem right.

Reply:
Maybe it's fine? What values does `p["julia_version"]` cover?

Reply:
https://github.com/JuliaPackaging/Yggdrasil/blob/5a363ee835394b70dcc268fe958f5c8e5cee912b/S/SLICOT/build_tarballs.jl#L103